src/main/webapp/app/exercises/text/participate/text-result/text-result-block.ts

```diff
@@ -29,7 +29,7 @@ export class TextResultBlock {
     }
 
     get feedbackType(): FeedbackType {
-        if (!this.feedback || this.feedback.credits == undefined || Feedback.isEmpty(this.feedback)) {
+        if (!this.feedback || this.feedback.credits === undefined || Feedback.isEmpty(this.feedback)) {
```

Comment:
Please revert this change. It is intended to be `== undefined`. Codacy will not report this anymore in the latest develop
```suggestion
        if (!this.feedback || this.feedback.credits == undefined || Feedback.isEmpty(this.feedback)) {
```

Reply:
But my local setup was always complaining that this is an error and wouldn't start the client

Reply:
fixed

Reply:
Yes, this was due to prettier complaining about your code style. This is fixed with the latest develop merge. :)